src/status_im/chat/events/requests.cljs

```diff
@@ -40,4 +40,4 @@
                  :status     "open"}]
     (-> fx
         (assoc-in [:db :chats chat-id :requests message-id] request)
-        (assoc ::save-request request))))
+        (update :save-entities #(conj (or % []) [:request request])))))
```

Comment:
Why do we have this generic `:save-entities` effect, what't the advantage, when we have to later do multimethod dispatch based on type ?

Reply:
Because doing this allows us to make sure that we don't perform two or more Realm write simultaneously.

Reply:
We never save more entities of the type `:chat`, `:request` or `:message` simultaneously, except for the special case, when we "receive" multiple console messages at once as part of the initial sign-up process - and that's process (onboarding) will be significantly changed anyway very soon, so building any kind of special logic (complicated by multimethods) around that seems to be not the best choice.

If you really wan't do go with plural effects (saving more messages at once), please do, but do not complicated it with multimethod dispatch, where in almost all cases we have to attach tag indicating the type of entity anyway, just create `:save-messages`/`:save-requests` etc additional effects (still keeping the original `:save-message`/`:save-request` effects which will be used almost always)